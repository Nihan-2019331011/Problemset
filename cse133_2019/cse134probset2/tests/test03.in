100
9601
6063
5193
3283
3695
6381
1448
6471
1700
5237
9160
9755
5833
4315
5947
7849
3163
3249
8942
9835
4002
9336
1094
7964
6748
4544
6148
2397
8393
8297
9395
9801
8989
5139
8614
6877
7293
5659
1533
2462
5853
9402
1308
4088
4421
8718
6436
9387
8411
9850
1007
9565
5594
8634
4818
5160
9202
7293
5081
8926
7749
7278
9795
7483
9576
7657
2307
6195
4616
1033
2677
2677
5882
9057
4763
8105
5773
7378
7925
9324
9633
9438
6874
1202
6584
3854
9308
4624
3911
6798
7629
2199
5621
9188
3933
1973
9355
1012
2261
3499
