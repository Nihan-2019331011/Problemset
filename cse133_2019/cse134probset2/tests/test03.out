201
145
131
88
100
151
49
152
56
132
194
204
142
116
145
175
85
86
190
205
105
196
42
177
157
120
147
74
183
182
197
205
191
131
186
162
166
141
52
74
143
197
47
108
117
187
151
197
183
205
37
201
138
187
126
131
194
166
130
190
174
166
205
170
201
174
71
147
121
39
77
77
145
194
124
179
142
169
176
196
202
199
162
43
154
103
196
121
103
160
174
69
138
194
105
62
197
37
70
95
